Filing UI not displaying FED alteration

lekshmimallika-aot opened the pull request:
*Issue #:* /bcgov/entity#11202

*Description of changes:*
The data section is available in the filings json only for a completed filing. Added a check for that

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

severinbeauvais commented:
> _Issue #:_ /[bcgov/entity#11202](https://github.com/bcgov/entity/issues/11202)
> 
> _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that
> 
> By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

Could you please post a screenshot of before and after? Or broken and fixed? I'm trying to understand what the expected details are.

severinbeauvais commented on src/components/Dashboard/FilingHistoryList.vue:
```diff
@@ -556,10 +556,12 @@ export default class FilingHistoryList extends Mixins(
-        item.isArrangement = this.isEffectOfOrderPlanOfArrangement(filing.data.order?.effectOfOrder)
-        item.toLegalType = filing.data.alteration?.toLegalType || null
-        item.fromLegalType = filing.data.alteration?.fromLegalType || null
+        if (item.isCompletedAlteration) {
```
How about a comment saying that this data is only valid for a completed alteration?

lekshmimallika-aot replied:
Sure

lekshmimallika-aot commented:
> > _Issue #:_ /[bcgov/entity#11202](https://github.com/bcgov/entity/issues/11202)
> > _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that
> > By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).
> 
> Could you please post a screenshot of before and after? Or broken and fixed? I'm trying to understand what the expected details are.

![broken](https://user-images.githubusercontent.com/8199923/154570616-c360ff67-5078-4d3e-9778-b7f7e0f20ebd.jpg)
![fixed](https://user-images.githubusercontent.com/8199923/154570641-ac11f675-5061-48d1-8c3c-94dccfcd7207.jpg)

severinbeauvais commented:
> _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that

Are you saying that attempting to display data that wasn't there caused the FED alteration to not display at all?

lekshmimallika-aot commented:
> > _Description of changes:_ The data section is available in the filings json only for a completed filing. Added a check for that
> 
> Are you saying that attempting to display data that wasn't there caused the FED alteration to not display at all?

Ya

cameron-freshworks reviewed (approved):
Short and sweet, gj!
